attalos/util/transformers/newnaivew2v.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+
+import attalos.util.log.log as l
+from attalos.imgtxt_algorithms.correlation.correlation import construct_W
+from attalos.util.transformers.transformer import Transformer
+
+logger = l.getLogger(__name__)
+
+
+class NaiveW2V(Transformer):
+    def __init__(self, w_arr):
+        self.w_arr = w_arr
+        
+    def transform(self, multihot_arr):
+        return np.dot(multihot_arr, self.w_arr.T)
+    
+    @staticmethod
+    def to_multihot(w2v_model, one_hot, predictions, k=5):
+        interpreted = []
+        logger.info("Converting predictions to multihot.")
+        for row_idx, prediction in enumerate(predictions):
+            if row_idx % 100 == 0:
+                logger.info("%s of %s." % (row_idx, len(predictions)))
+            predicted_words = w2v_model.closest_words(prediction, k)
```

Comment:
Since we are replacing the call to NaiveW2V.to_multihot in `attalos/imgtxt_algorithms/approaches/naivesum.py` to replicate negsampling.py, I'm going to leave this alone.